apps/src/templates/rubrics/RubricSettings.jsx

```diff
@@ -231,6 +231,16 @@ export default function RubricSettings({
     setStatusAll(STATUS_ALL.EVALUATION_PENDING);
     const url = `/rubrics/${rubricId}/run_ai_evaluations_for_all`;
     const params = {section_id: sectionId};
+    const eventName = EVENTS.TA_RUBRIC_SECTION_AI_EVAL;
+    analyticsReporter.sendEvent(
+      eventName,
+      {
+        ...(reportingData || {}),
+        rubricId: rubricId,
+        sectionId: sectionId,
+      },
+      PLATFORMS.BOTH
+    );
```

Comment:
Just to double check, you mean that we want an event for each student in this loop right before the evaluate rubric job is called?: https://github.com/code-dot-org/code-dot-org/blob/d5ed4e18f86377b9d0ad83fa12ae94d60784dd96/dashboard/app/controllers/rubrics_controller.rb#L196-L200
Is this event in rubric settings still useful, or should it be removed? Does the event for the button click for an individual student need to be moved to the backend as well?

Reply:
> Just to double check, you mean that we want an event for each student in this loop right before the evaluate rubric job is called?:

correct!

> Is this event in rubric settings still useful, or should it be removed? 

it could be useful, so since it is already implemented let's keep it.

> Does the event for the button click for an individual student need to be moved to the backend as well?

let's keep this one in the frontend, since it could be useful to have this in amplitude too.

Reply:
> > Is this event in rubric settings still useful, or should it be removed?
> 
> it could be useful, so since it is already implemented let's keep it.

if you keep it, it would be handy to have a different event name, so that we can easily distinguish between the teacher doing a thing and the individual student project being evaluated.